render.yaml

```diff
@@ -20,6 +21,10 @@ services:
           property: hostport
       - key: HASURA_GRAPHQL_SERVER_PORT
         value: 10000
+      - key: HASURA_GRAPHQL_ENABLE_CONSOLE
+        value: true
+      - key: PORT
```

Comment:
Do we still need the other port config also?

Reply:
Yeah, `HASURA_GRAPHQL_SERVER_PORT` has no effect on the setting and without it the instance isn't reachable.